src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -272,11 +272,16 @@ such as `stream:true` or `max_tokens:5`. This flag can be repeated to supply mul
 
 
 
-##### `--endpoint {v1/completions,v1/chat/completions}`
+##### `--endpoint {completions,chat}`
 
 Describes what endpoint to send requests to on the server. This is required when
 using `openai` service-kind. This is ignored in other cases.
 
+##### `--port <str>`
```

Comment:
Is port the right terminology for this purpose? Port makes me think of the [port number](https://en.wikipedia.org/wiki/Port_(computer_networking)) and I wonder if this could cause confusion to the users.

Reply:
This is what izzy asked for originally but matt and I already talked about revisiting it

Reply:
Updated endpoint to be api: more directly reflects what we are using, which is an openai api.
Updated port to endpoint: port is the wrong term here. We were basically defining a custom endpoint variable.

Reply:
Moved to --endpoint-type instead for api based on offline discussion.